tests/integration/components/frost-object-browser-test.js

```diff
@@ -122,11 +129,23 @@ describeComponent(
 
     it('it changes page when we click to next change button', function () {
       this.timeout(8000)
-      this.render(hbs`{{frost-object-browser
-        itemsPerPage=6
-        values=model.resources
-        model=model.model
-      }}`)
+
+      this.render(hbs`
+        {{#frost-object-browser
+          itemsPerPage=6
```

Comment:
Do we need this on the object browser interface if it's part of the yielded pagination interface?  I can see value in keeping it on the object browser (common to both pagination and infinite scroll), but in that case we should set the attr directly on the frost-object-browser-paginator and yield that component back to the pagination slot so that people don't need to deal with that boilerplate (same for computedPageNumber and computedValuesTotal)